uc_reports/uc_reports.admin.inc

```diff
@@ -663,9 +730,14 @@ function uc_reports_products_custom_form($form, &$form_state, $values) {
  * @see uc_reports_products_custom_form_submit()
  */
 function uc_reports_products_custom_form_validate($form, &$form_state) {
-  if (empty($form_state['values']['status'])) {
+  $statuses = array_filter($form_state['values']['status']);
+  if (empty($statuses)) {
```

Comment:
Why not make this option required and skip these checks (here and below)?

Reply:
Yup, makes sense. Adding `'#required' => TRUE` lets us eliminate the `...form_validate()`. (Only reason for doing it this way was that's how it was originally.)